lib/dagnabbit.py

```diff
@@ -133,50 +136,80 @@ def parse_dagnabbit(
+                    of.write(
+                        f'VARS {name} JOBSUBJOBSECTION="{count}" CM2="{count-2}" CM1="{count-1} nodename="$(JOB)"\n'
+                    )
+
+                else:
+
+                    prev_jobsub_line = line
+                    prev_jobsub_count = count
+
+                    parser = get_parser()
+                    try:
+                        line_argv = line.strip().split()[1:]
+                        backslash_escape_layer(line_argv)
+                        res = parser.parse_args(line_argv)
+                    except:
+                        sys.stderr.write(
+                            f"Error at file {values['dag']} line {linenum}\n"
+                        )
+                        sys.stderr.write(f"parsing: {line.strip().split()}\n")
+                        sys.stderr.flush()
+                        raise
+                    # print(f"vars(res): {repr(vars(res))}")
+                    # handle -f drobpox: etc. in dag stages
+                    do_tarballs(res)
+                    thesevalues = values.copy()
+                    thesevalues["N"] = 1
+                    thesevalues["dag"] = None
+                    # don't take executable from command line, only from DAG file
+                    if "full_executable" in thesevalues:
+                        del thesevalues["full_executable"]
+                    if "executable" in thesevalues:
+                        del thesevalues["executable"]
+
+                    # we get a bunch of defaults from the command line parser that
+                    # we don't want to override from the initial command line
+                    update_with: Dict[str, Any] = vars(res)
+                    kl = list(update_with.keys())
+                    for k in kl:
+                        if update_with[k] is parser.get_default(k):
+                            del update_with[k]
+
+                    thesevalues.update(update_with)
+                    set_extras_n_fix_units(thesevalues, schedd_name, proxy, token)
+                    thesevalues["script_name"] = f"{name}.sh"
+                    thesevalues["cmd_name"] = f"{name}.cmd"
+                    with open(
+                        os.path.join(dest, f"{name}.cmd"), "w", encoding="UTF-8"
+                    ) as cf:
+                        cf.write(
+                            jinja_env.get_template("simple.cmd").render(**thesevalues)
+                        )
+                    with open(
+                        os.path.join(dest, f"{name}.sh"), "w", encoding="UTF-8"
+                    ) as csf:
+                        csf.write(
+                            jinja_env.get_template("simple.sh").render(**thesevalues)
+                        )
+                    of.write(f"\nJOB {name} {name}.cmd\n")
+                    of.write(
+                        f'VARS {name} JOBSUBJOBSECTION="{count}" CM2="{count-2}" CM2="{count-1}" nodename="$(JOB)"\n'
```

Comment:
Shouldn't this be `CM2="{count -2}" CM1="{count-1}"` like above (line 153)?  I see two `CM2`'s in this line.

Reply:
Ack!  yes.